js/faq.test.js

```diff
@@ -0,0 +1,77 @@
+const removeFocusForOthers = require('./faq');
+
+describe('handle keydown event', () => {
+  let ancTag, event;
+  beforeEach(() => {
+    ancTag = document.createElement('a');
+    ancTag.setAttribute('href', '/example');
+    ancTag.click = jest.fn();
+    document.body.appendChild(ancTag);
```

Comment:
do we not need to define `document`. will it understand `dom`  automatically ?

Reply:
It will take **DOM** automatically